<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxDoc">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>241</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>Document</string>
    </property>
    <widget class="QPushButton" name="btnAgoraSite">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>20</y>
       <width>235</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Document Website</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFAQ">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>50</y>
       <width>235</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>FAQ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRegister">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>80</y>
       <width>235</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Register Account</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGithub">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>110</y>
       <width>235</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Github</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxBasic">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>170</y>
      <width>241</width>
      <height>155</height>
     </rect>
    </property>
    <property name="title">
     <string>Basic Scene</string>
    </property>
    <widget class="QListWidget" name="listWidgetBasic">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>20</y>
       <width>235</width>
       <height>130</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxAdvanced">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>345</y>
      <width>241</width>
      <height>375</height>
     </rect>
    </property>
    <property name="title">
     <string>Advanced Scene</string>
    </property>
    <widget class="QListWidget" name="listWidgetAdvanced">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>20</y>
       <width>235</width>
       <height>350</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxScene">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>951</width>
      <height>711</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1213</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
